nucleus/pom.xml

```diff
@@ -160,7 +160,7 @@
         <mimepull.version>1.9.12</mimepull.version>
 
         <!-- Apache Felix is an open source implementation of the OSGi Core Release 6 framework specification. -->
-        <org.apache.felix.main.version>6.0.2</org.apache.felix.main.version>
```

Comment:
There are several more updated  Felix subprojects. Shouldn't their references be updated too?

Reply:
I will upgrade the subprojects.